### PR TITLE
Improved Gunit's task implementation

### LINKED ISSUE #15: Create Task and integrate with Backlog items for project

### LINKED ISSUE #48: As a user, I can specify the tasks needed to be done for each new backlog item.

### LINKED ISSUE #49: As a user, I can specify the name of the tasks.

### LINKED ISSUE #50: As a user, I can delete tasks, so that tasks that are no longer viable does not clutter the list.

### LINKED ISSUE #51: As a user, I am able to mark these tasks done/subtasks, so that I know what's done and what's left to do.

### LINKED ISSUE #52: As a user, I am able to create a list of tasks which contains everything that need to be delivered by the team for the final product.

### LINKED ISSUE #54: As a user, I can set the priority level for each task, so that the more important tasks are prioritized.

### LINKED ISSUE #55: As a user, I am able to reprioritize the tasks in my list, move them up and down in order of priority and need.

### PR DESCRIPTION
**Pull request by e0426051**
Fixed some bugs, improved UI messages, implement TaskRunner features where applicable.
No test/JUnit implemented in this PR.

Fixed Issue #15

### CODE REVIEW

File: src/main/java/seedu/duke/command/TaskCommand.java [hunk 1]
```diff
@@ -38,106 +38,121 @@ public void addTaskCommand(Hashtable<String, String> tasks, Ui ui, ArrayList<Pro
...
-        Project proj = projectList.get(0);
-        Task task = new Task(title, description, priority);
-        proj.backlog.addTask(task);
+        try {
+            Project proj = projectList.get(0);
+            Task task = new Task(title, description, priority);
+            proj.backlog.addTask(task);
+
+            ui.printTaskAdded(proj.backlog.backlogTasks.get(proj.backlog.size() - 1));
+        } catch (IndexOutOfBoundsException e) {
```
Comment (gmit22) [inline src/main/java/seedu/duke/command/TaskCommand.java:48]: change this get to getTask, so it becomes backlog.getTask(id -1 )

File: src/main/java/seedu/duke/command/TaskCommand.java [hunk 2]
```diff
@@ -38,106 +38,121 @@ public void addTaskCommand(Hashtable<String, String> tasks, Ui ui, ArrayList<Pro
...
+        try {
+            Project proj = projectList.get(0);
+            Collections.sort(taskId); //sort array so that it deletes the largest number first
+            Collections.reverse(taskId); //but the remove print message is in reverse
+            for (String id : taskId) {
+                try {
+                    int backlogId = Integer.parseInt(id) - 1;
+                    if (backlogId < proj.backlog.size()) {
+                        ui.printTaskRemoved(proj.backlog.backlogTasks.get(backlogId));
+                        proj.backlog.backlogTasks.remove(backlogId);
```
Comment (gmit22) [inline src/main/java/seedu/duke/command/TaskCommand.java:65]: Similar to add, maybe implement a removeTask also in the backlog task class

**Comment by gmit22** [comment]
Suggested some changes, rest looks good to merge

**Comment by gmit22** [comment]
Please assign appropriate labels, and everything.

**Comment by e0426051** [comment]
Not merge-ready yet, found other places to implement getTask()

**Comment by e0426051** [comment]
Labels added and getTask() has been implemented for all applicable functions

**Comment by gmit22** [comment]
Good to merge

**Pull request merged**